src/EmbedIndex/EmbedIndex.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\FileFormats.ELF\FileFormats.ELF.csproj" />
+    <ProjectReference Include="..\FileFormats.MachO\FileFormats.MachO.csproj" />
+    <ProjectReference Include="..\FileFormats.PDB\FileFormats.PDB.csproj" />
+    <ProjectReference Include="..\FileFormats.PE\FileFormats.PE.csproj" />
+    <ProjectReference Include="..\FileFormats\FileFormats.csproj" />
+  </ItemGroup>
+
+  <!-- Workaround for https://github.com/Microsoft/msbuild/issues/1721. -->
+  <Import Project="..\Directory.Build.targets" Condition="'$(TargetFramework)' == ''" />
```

Comment:
Not necessary for projects that do not multi-target. Also, should be fixed in dotnet cli 2.0.